packages/client/hmi-client/src/components/models/tera-stratified-model-configuration.vue

```diff
@@ -153,6 +180,16 @@ const tableHeaders = computed<{ name: string; colspan: number }[]>(() => [
 	{ name: 'Parameters', colspan: baseModelTransitions.value.length }
 ]);
 
+// Makes cell inputs focus once they appear
+const vFocus = {
+	mounted: (el) => el.focus()
+};
+
+function updateModelConfigName(configIndex: number) {
+	modelConfigurations.value[configIndex].name = modelConfigInputValue.value;
+	updateModelConfiguration(modelConfigurations.value[configIndex]);
+}
+
```

Comment:
Could be worth putting this in a separate util/service? Might be out of scope for this PR but I know the `tera-model-configuration` uses a similar function.  I imagine both these components use similar function to edit names/values as well.  Just a note

Reply:
Yeah later I want to look into the viability of merging the two config components in some way. This component is made from a copy of the other config component - had to quickly get this version working during the hackathon.